app/main.py

```diff
@@ -34,45 +34,45 @@ class Bridge:
     def __init__(self):
         """Initialise a Bridge object."""
         self._rabbit_hosts = [
-            settings.RABBIT_HOST,
-            settings.RABBIT_HOST2
+            settings.SDX_GATEWAY_EQ_RABBITMQ_HOST,
+            settings.SDX_GATEWAY_EQ_RABBITMQ_HOST2
```

Comment:
`_rabbit_hosts` isn't consistent with the renaming of the other properties, I think it should be `_eq_queue_hosts`.